database/index.ts

```diff
@@ -10,8 +10,14 @@ import sequelizeConfig from "@database/config.js"
  * fake db models into sequelize, rather than depending upon concrete instances
  * in our unit tests.
  */
-export default (models: Parameters<Sequelize["addModels"]>[0]) => {
-  const sequelize = new Sequelize(sequelizeConfig as SequelizeOptions)
+export default (
+  models: Parameters<Sequelize["addModels"]>[0],
+  dbConfig?: SequelizeOptions
+) => {
+  const sequelize = new Sequelize({
+    ...sequelizeConfig,
+    ...dbConfig,
```

Comment:
Which sequelizeConfig vars do we expect to override with dbConfig? If it is database, user, password, port, host - could we instead modify the env vars for the test env?

Reply:
Yes - I was wondering if we could use a test env file like `.env.test`, but open to either.